apps/beeswax/src/beeswax/server/dbms.py

```diff
@@ -166,32 +175,47 @@ def get_query_server_config(name='beeswax', connector=None):
 
     elif name != 'hms' and name != 'impala':
       activeEndpoint = cache.get("hiveserver2")
+      zk = KazooClient(hosts=libzookeeper_conf.ENSEMBLE.get(), read_only=True)
```

Comment:
Here is feels like we are going to query ZooKeeper even when not using the Hive discovery? i.e. it will brake regular usage?

Reply:
yup... created a function and moved it there so we just call the function now